main/src/library/Int32.flix

```diff
@@ -102,38 +102,38 @@ namespace Int32 {
     /// A zero is shifted into the leftmost position regardless of sign extension.
     ///
     @Time(1) @Space(1)
-    pub def logicalRightShift(x: Int32, distance: Int32): Int32 =
-        if (x < 0 and distance rem size() != 0)
-            ((x >>> 1) &&& maxValue()) >>> (distance - 1)
+    pub def logicalRightShift(distance: {distance :: Int32}, x: Int32): Int32 =
```

Comment:
Since both arguments are `Int32` i made `distance` a record. Do we want the whole word or something like `dist`?

Reply:
Yes, lets do it!